app/Http/Controllers/Api/UsersController.php

```diff
@@ -74,6 +83,17 @@ public function store(UserRequest $request)
         }
 
         $contest->waitingRoom->users()->attach($user->id);
+        $this->manager->appendCampaign($contest);
+
+        // Fire off welcome Email
+        $message = Message::where(['contest_id' => $contest->id, 'type' => 'Welcome'])->first();
```

Comment:
I *think* `Welcome` should be lowercased...that's how it's stored in the db at least.